### PR TITLE
[Feature]:Flink-connector supports streamload parameters

### LINKED ISSUE #6199: [Feature]:flink-connector supports streamload parameters
**Is your feature request related to a problem? Please describe.**

flink-connector supports parameters of streamload tasks
streamload parameters ：http://doris.apache.org/master/zh-CN/administrator-guide/load-data/stream-load-manual.html#%E5%88%9B%E5%BB%BA%E5%AF%BC%E5%85%A5

### PR DESCRIPTION
**Pull request by JNSimba**
## Proposed changes
Flink-connector supports streamload parameters
#6199 
## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
- [ ] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [ ] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #6199 ) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: extension/flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisDynamicTableFactory.java [hunk 1]
```diff
@@ -234,27 +239,39 @@ private DorisReadOptions getDorisReadOptions(ReadableConfig readableConfig) {
 		return builder.build();
 	}
 
-	private DorisExecutionOptions getDorisExecutionOptions(ReadableConfig readableConfig) {
+	private DorisExecutionOptions getDorisExecutionOptions(ReadableConfig readableConfig,Properties streamLoadProp) {
```
Comment (pengxiangyu) [inline extension/flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisDynamicTableFactory.java:242]: need a space before Properties

File: extension/flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisDynamicTableFactory.java [hunk 2]
```diff
@@ -234,27 +239,39 @@ private DorisReadOptions getDorisReadOptions(ReadableConfig readableConfig) {
...
+		builder.setStreamLoadProp(streamLoadProp);
 		return builder.build();
 	}
 
+	private Properties getStreamLoadProp(Map<String, String> tableOptions){
+		final Properties streamLoadProp = new Properties();
+
+		for(Map.Entry<String,String> entry : tableOptions.entrySet()){
+			if(entry.getKey().startsWith(STREAM_LOAD_PROP_PREFIX)){
+				String subKey = entry.getKey().substring((STREAM_LOAD_PROP_PREFIX).length());
```
Comment (pengxiangyu) [inline extension/flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisDynamicTableFactory.java:256]: STREAM_LOAD_PROP_PREFIX need not ( )

File: extension/flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisDynamicTableFactory.java [hunk 3]
```diff
@@ -234,27 +239,39 @@ private DorisReadOptions getDorisReadOptions(ReadableConfig readableConfig) {
...
-		helper.validate();
+		helper.validateExcept(STREAM_LOAD_PROP_PREFIX);
+
+		Properties streamLoadProp = getStreamLoadProp(context.getCatalogTable().getOptions());
 		// create and return dynamic table source
 		return new DorisDynamicTableSink(
 				getDorisOptions(helper.getOptions()),
 				getDorisReadOptions(helper.getOptions()),
-				getDorisExecutionOptions(helper.getOptions())
+				getDorisExecutionOptions(helper.getOptions(),streamLoadProp)
```
Comment (pengxiangyu) [inline extension/flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisDynamicTableFactory.java:274]: need a space before streamLoadProp

File: extension/flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisStreamLoad.java [hunk 4]
```diff
@@ -54,15 +57,17 @@
     private String db;
     private String tbl;
     private String authEncoding;
+    private Properties streamLoadProp;
 
-    public DorisStreamLoad(String hostPort, String db, String tbl, String user, String passwd) {
+    public DorisStreamLoad(String hostPort, String db, String tbl, String user, String passwd,Properties streamLoadProp) {
```
Comment (pengxiangyu) [inline extension/flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisStreamLoad.java:62]: space

File: extension/flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisStreamLoad.java [hunk 5]
```diff
@@ -89,6 +94,9 @@ private HttpURLConnection getConnection(String urlStr, String label) throws IOEx
         conn.addRequestProperty("Expect", "100-continue");
         conn.addRequestProperty("Content-Type", "text/plain; charset=UTF-8");
         conn.addRequestProperty("label", label);
+        for(Map.Entry<Object, Object> entry : streamLoadProp.entrySet()){
+            conn.addRequestProperty(String.valueOf(entry.getKey()),String.valueOf(entry.getValue()));
```
Comment (pengxiangyu) [inline extension/flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisStreamLoad.java:98]: space

File: extension/flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisDynamicOutputFormat.java [hunk 6]
```diff
@@ -66,21 +74,23 @@ public DorisDynamicOutputFormat(DorisOptions option,DorisReadOptions readOptions
         this.options = option;
         this.readOptions = readOptions;
         this.executionOptions = executionOptions;
+        this.fieldDelimiter = executionOptions.getStreamLoadProp().getProperty(FIELD_DELIMITER_KEY,FIELD_DELIMITER_DEFAULT);
+        this.lineDelimiter = executionOptions.getStreamLoadProp().getProperty(LINE_DELIMITER_KEY,LINE_DELIMITER_DEFAULT);
```
Comment (pengxiangyu) [inline extension/flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisDynamicOutputFormat.java:78]: space

File: extension/flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisStreamLoad.java [hunk 7]
```diff
@@ -133,11 +141,15 @@ public void load(String value) throws StreamLoadException {
...
     private LoadResponse loadBatch(String value) {
-        Calendar calendar = Calendar.getInstance();
-        String label = String.format("flink_connector_%s%02d%02d_%02d%02d%02d_%s",
-                calendar.get(Calendar.YEAR), calendar.get(Calendar.MONTH) + 1, calendar.get(Calendar.DAY_OF_MONTH),
-                calendar.get(Calendar.HOUR_OF_DAY), calendar.get(Calendar.MINUTE), calendar.get(Calendar.SECOND),
-                UUID.randomUUID().toString().replaceAll("-", ""));
+        String label = streamLoadProp.getProperty("label");
+        if(StringUtils.isBlank(label)){
+            Calendar calendar = Calendar.getInstance();
+            label = String.format("flink_connector_%s%02d%02d_%02d%02d%02d_%s",
```
Comment (pengxiangyu) [inline extension/flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisStreamLoad.java:147]: SimpleDateFormat is easiler to use and read

File: extension/flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisDynamicTableFactory.java (hunk 1 above)
Comment (JNSimba) [reply extension/flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisDynamicTableFactory.java:242]: Okay, the code style will be optimized

File: extension/flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisStreamLoad.java (hunk 7 above)
Comment (JNSimba) [reply extension/flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisStreamLoad.java]: already modified

**Review by pengxiangyu — APPROVED** [review]
LGTM

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**